raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -1,4 +1,6 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
+import { makeHash } from '../integration/mocks';
```

Comment:
We shouldn't import from integration mocks. That pulls in some side-effect mocks as well (`patch*` functions). Please, create a `tests/utils.ts` module or alike, and pull up that function. There're some others which need to go there as well, like `makeAddress`.

Reply:
Done, please have a look. I'm not sure if the `from '../utils';` import is correct.